lib/beaker/hypervisor/vmpooler.rb

```diff
@@ -215,6 +215,35 @@ def provision
       end
 
       @logger.notify 'Spent %.2f seconds tagging VMs' % (Time.now - start)
+
+      # add additional disk to vm
+      @logger.notify 'Looking for disks to add...'
+
+
+      @hosts.each do |h|
+        hostname = h['vmhostname'].split(".")[0]
+
+        if h['disks']
+          @logger.debug "Found disks for #{hostname}!"
+          disks = h['disks']
+
+          disks.each_with_index do |disk_size, index|
+            start = Time.now
+
+            response = add_disk(hostname, disk_size)
+            parsed = parse_response(response)
+
+            done = wait_for_disk(hostname, disk_size, index)
+            if done
+              @logger.notify "Spent %.2f seconds adding disk #{index}. " % (Time.now - start)
+            else
+              @logger.notify 'Timeout, cannot verify disk was added.'
```

Comment:
Does this mean that beaker might not be able to verify that the disks were added, but still continue with the test run?